webapp/src/main/java/io/github/microcks/web/TestController.java

```diff
@@ -174,6 +178,38 @@ public ResponseEntity<TestCaseResult> reportTestCaseResult(@PathVariable("id") S
       return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
    }
 
+   @GetMapping("/test-fetch")
+   public ResponseEntity<String> testFetch() {
+      return ResponseEntity.ok("Hello from test-fetch endpoint!");
+   }
+
+   @PostMapping("/test-fetch")
+   public ResponseEntity<String> testFetchPost(@RequestBody(required = false) String body) {
+      return ResponseEntity.ok("POST received with body: " + (body != null ? body : "empty"));
+   }
+
+   @PutMapping("/test-fetch")
+   public ResponseEntity<String> testFetchPut(@RequestBody(required = false) String body) {
+      return ResponseEntity.ok("PUT received with body: " + (body != null ? body : "empty"));
+   }
+
+   @DeleteMapping("/test-fetch")
+   public ResponseEntity<String> testFetchDelete() {
+      return ResponseEntity.ok("DELETE received");
+   }
+
+   @PatchMapping("/test-fetch")
+   public ResponseEntity<String> testFetchPatch(@RequestBody(required = false) String body) {
+      return ResponseEntity.ok("PATCH received with body: " + (body != null ? body : "empty"));
+   }
+
+   @GetMapping("/test-fetch-headers")
+   public ResponseEntity<String> testFetchHeaders(HttpServletRequest request) {
+      String customHeader = request.getHeader("X-Custom-Header");
+      String authHeader = request.getHeader("Authorization");
+      return ResponseEntity.ok("Headers received - Custom: " + customHeader + ", Auth: " + authHeader);
+   }
+
```

Comment:
I think we have to relocate this to another Controller/REST resource as `TestController` is dedicated to Microcks' TestResult entities management. Couldn't we put this in a dedicated controller under `/src/test/java` instead?

Reply:
done here: https://github.com/microcks/microcks/pull/1656/commits/ab3923398a23c4bd81b381a5b6b3ba41bcd1bc1e